packages/app-project/src/shared/components/Stat/Stat.spec.js

```diff
@@ -1,27 +1,19 @@
 import { render, screen } from '@testing-library/react'
 import { composeStory } from '@storybook/react'
-import { format } from 'd3'
-import Meta, { Default, HugeNumber, Zero } from './Stat.stories.js'
-import * as Mocks from './Stat.mock'
+import Meta, { Default } from './Stat.stories.js'
+import { DefaultMock  } from './Stat.mock'
 
+/* Note that this UI is animated, so we cannot look for values in AnimatedNumber
+components via unit test environment */
 
 describe('Component > Stat', function () {
-  [Default, HugeNumber, Zero].forEach(function (Story) {
-    describe(`${Story.name} Stat`, function () {
-      let mock = Mocks[`${Story.name}Mock`]
+  const DefaultStory = composeStory(Default, Meta)
```

Comment:
Changed this to a unit test for only one story. Every story has a label, so we only need to test it once.